index.js

```diff
@@ -61,7 +61,16 @@ say.speak = function(text, voice, speed, callback) {
     pipedData += '(SayText \"' + text + '\")';
   } else if (process.platform === 'win32') {
     pipedData = text;
-    commands = [ 'Add-Type -AssemblyName System.speech; $speak = New-Object System.Speech.Synthesis.SpeechSynthesizer; $speak.Speak([Console]::In.ReadToEnd())' ];
+
+    commands = ['–NoProfile –ExecutionPolicy Restricted;', 
+    'Add-Type -AssemblyName System.speech;', 
+    '$speak = New-Object System.Speech.Synthesis.SpeechSynthesizer;'];
+    try {        
+        var rate = parseInt(speed);
+        commands.push('$speak.rate = ' + (between(rate, -10, 10) ? 0 : rate).toString() + ';');
```

Comment:
I believe your between ternary is backwards. It's saying if the value is valid, use 0, and if invalid, use the invalid value.